src/main/java/org/allenai/wikitables/WikiTablesExample.java

```diff
@@ -0,0 +1,44 @@
+package org.allenai.wikitables;
+
+import java.util.Set;
+import java.util.List;
+import java.util.ArrayList;
+
+import com.jayantkrish.jklol.ccg.DependencyStructure; 
+import com.jayantkrish.jklol.ccg.CcgExample;
+import com.jayantkrish.jklol.ccg.lambda2.Expression2; 
+import com.jayantkrish.jklol.nlpannotation.AnnotatedSentence;
+
+/**
+ * Similar to CcgExample, except that this supports multiple logcal forms per example.
+ * This correctly does not hold a syntactic parse or dependencies. But we may need them in the future.
+ */
+
+public class WikiTablesExample extends CcgExample {
```

Comment:
This shouldn't extend CcgExample. Just add a field to store the AnnotatedSentence.